events/blocks/mcz-handler/mcz-handler.js

```diff
@@ -0,0 +1,73 @@
+import { CheckResourceLocation } from '../../../rs/360-KCI-804/images/mktoTestFormConfig.js';
+import { setMetadata, getMetadata } from '../../scripts/utils.js';
+
+export default async function init(el) {
+  const rows = Array.from(el.children);
+  const resourceLocation = `#${rows[0].textContent.trim().toLowerCase()}`;
+  // suggestion to use getElementbyId
+  const key = rows[1].textContent.trim().toLowerCase();
+
+  const resourceWatch = 'main .section .chrono-box';
+
+  el.innerHTML = '';
+
+  el.setAttribute('data-mcz-dl-status', 'loading');
+
+  let mczId = null;
+  if (getMetadata('eventExternalId')) {
```

Comment:
<img width="192" height="62" alt="Image" src="https://github.com/user-attachments/assets/373819c9-8447-4430-bd0c-6e143a1b9b3f" />

The column currently in the metadata sheet is called 'external-event-id'. Your code is looking for eventExternalId. Is this intended? Are you planning on using the existing column or adding a new column? Also, I think generally we don't have camelCase metadata.